packages/meditrak-app/app/entityMenu/EntityList.js

```diff
@@ -120,6 +121,22 @@ export class EntityList extends PureComponent {
     });
   };
 
+  getListSections = () => {
+    const { searchResults } = this.state;
+    if (searchResults) {
+      return [{ data: searchResults }];
+    }
+
+    const { entities, recentEntities } = this.props;
+    if (recentEntities && recentEntities.length > 0) {
```

Comment:
```suggestion
    if (recentEntities?.length > 0) {
```
I'm not sure if mobile can use this but maybe it can?

Reply:
I had the idea it couldn't, but just tried and it works!